src/components/dashboard/Overview/Overview.tsx

```diff
@@ -180,12 +200,16 @@ const Overview = (): ReactElement => {
               </Grid>
             </Grid>
             <Grid item mt="auto">
-              <Grid container mt={3} spacing={1} flexWrap="wrap">
-                <Grid item xs={12} sm="auto">
-                  <BuyCryproButton />
-                </Grid>
-
-                <Grid item xs={6} sm="auto">
+              <Grid
+                container
+                mt={3}
+                spacing={1}
+                flexWrap="wrap"
+                style={{ display: 'flex', justifyContent: 'space-between' }}
+              >
+                <BuyCryproButton />
```

Comment:
I would move `<Grid item style={{ flex: 1 }}>` here, instead of inside the `BuyCryproButton`.

Reply:
Unfortunately, it is not possible, because we control show/hide logic inside of the button, when we put `<Grid />` wrapper outside the button it will hold the space and move other buttons in Overview widget

Reply:
Ah, true, thanks!